book.toml

```diff
@@ -48,3 +48,6 @@ editable = true
 "generics/closures.html" = "traits/closures.html"
 "generics/impl-trait.html" = "traits/impl-trait.html"
 "generics/trait-bounds.html" = "traits/trait-bounds.html"
+
+[output.exerciser]
+output-directory = "exercise-templates"
```

Comment:
Could we use a name which is more distinct for both the top-level folder inside the zip file _and_ the zip file itself?

I would suggest `comprehensive-rust-exercises` for both.

Reply:
Done.